TrackerUI/MainWindow.xaml.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Threading;
+using System.Diagnostics;
+using System.IO;
+using nsTracker;
+using nsTask;
+
+namespace TrackerUI
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+        }
+
+        private void Window_Loaded(object sender, RoutedEventArgs e)
+        {
+            StartTimer();
+            LoadComboboxItems();
+        }
+
+        private void LoadComboboxItems()
+        {
+            foreach (var item in Properties.Settings.Default.Combobox)
+            {
+                SelectionCombobox.Items.Add(item.ToUpper());
+            }
+        }
+
+        private void StartTimer()
```

Comment:
I think this is not a Timer, this is the current date and time that it is appearing in the app. Maybe you should rename it.